Civ6Client.py

```diff
@@ -6,15 +6,15 @@
 import zipfile
 
 from CommonClient import ClientCommandProcessor, CommonContext, get_base_parser, logger, server_loop, gui_enabled
-from .Data import get_progressive_districts_data
-from .DeathLink import handle_check_deathlink
+from worlds.civ_6.Data import get_progressive_districts_data
+from worlds.civ_6.DeathLink import handle_check_deathlink

```

Comment:
Put it back in your world -- main doesn't want more exes I think

Reply:
Ah will do, good to know

Reply:
Ok I just copied the styling from [here](https://github.com/hesto2/Archipelago/blob/civ6-1.0/worlds/_bizhawk/client.py#L16). Lmk if that works or if I'm missing a beat here 👍

Reply:
Looks generally fine? As long as it functions o:

Reply:
Yeah verified it same way w/ dropping my `.apcivvi` file on it and also clicking it in the list 🎉